README.md

```diff
@@ -42,3 +42,9 @@ You can run tests in `wp-env` with the following command:
 ```shell
 composer dev:test
 ```
+
+If you want to run only one test, you can use the following command:
+
+```shell
+composer dev:test -- --filter methods_name
```

Comment:
FYI, you can also do `--filter Class_Name` to run all the tests of a specific class/file.